sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Testing/LiveResourceManager.cs

```diff
@@ -30,241 +30,281 @@ public sealed class LiveResourceManager
         /// <summary>The maximum number of attempts to retry a management operation.</summary>
         public const int RetryMaximumAttempts = 20;
 
-        /// <summary>The number of seconds to use as the basis for backing off on retry attempts.</summary>
-        private const double RetryExponentialBackoffSeconds = 3.0;
-
-        /// <summary>The number of seconds to use as the basis for applying jitter to retry back-off calculations.</summary>
-        private const double RetryBaseJitterSeconds = 60.0;
-
-        /// <summary>The buffer to apply when considering refreshing; credentials that expire less than this duration will be refreshed.</summary>
-        private static readonly TimeSpan CredentialRefreshBuffer = TimeSpan.FromMinutes(5);
+        /// <summary>The length of time to use as the basis for backing off on retry attempts.</summary>
+        private readonly TimeSpan RetryExponentialBackoff = TimeSpan.FromSeconds(3);
 
-        /// <summary>The random number generator to use for each requesting thread.</summary>
-        private static readonly ThreadLocal<Random> RandomNumberGenerator = new ThreadLocal<Random>(() => new Random(Interlocked.Increment(ref s_randomSeed)), false);
-
-        /// <summary>The seed to use for random number generation.</summary>
-        private static int s_randomSeed = Environment.TickCount;
-
-        /// <summary>The token credential to be used with the Event Hubs management client.</summary>
-        private static ManagementToken s_managementToken;
+        /// <summary>The maximum length of time allow for backing off on retry attempts.</summary>
+        private readonly TimeSpan RetryMaximumBackoff = TimeSpan.FromMinutes(20);
```

Comment:
Just wanted to be sure this is meant to be 20 minutes and not 20 seconds.

Reply:
Oh, yes.... that's intentional.   😄